pkg/hintrunner/zero/zerohint_memcpy.go

```diff
@@ -3,9 +3,59 @@ package zero
 import (
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/core"
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+
 	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
+	f "github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
+func newMemcpyContinueCopyingHint(continueCopying hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "MemcpyContinueCopying",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> n -= 1
+			//> ids.continue_copying = 1 if n > 0 else 0
+
+			//> n-=1
+			n, err := ctx.ScopeManager.GetVariableValue("n")
+			if err != nil {
+				return err
+			}
+
+			felt := new(f.Element)
```

Comment:
nitpick: this can be a oneliner ```n.Sub(n, &utils.FeltOne)```. You don't need felt variable then and can safely use ```n```

Reply:
Hey, @MaksymMalicki when trying this I found that is not possible 😢 when we try to do `n. Sub` `n` is of type `any`, and doesn't know the `.Sub` method, also I tried doing type casting but the operations didn't work as expected